README.md

```diff
@@ -172,7 +172,7 @@ cd ws_stream_tungstenite
 cargo run --example echo --release
 
 # in a different terminal:
-cargo run --example echo_tt --release -- "127.0.0.1:3312"
+cargo run --example echo_tt --release -- "ws://127.0.0.1:3312"
```

Comment:
But this one is for the server. It has to parse to `std::net::SocketAddr` so it must not have a protocol.

Reply:
Wait... oh... sorry, I changed the wrong line!!